tests/integration/docker/swarm.bats

```diff
@@ -44,37 +44,37 @@ setup() {
 	while [ `docker ps -a | tail -n +2 | egrep "Up [0-9]*"|  wc -l`  -lt  $number_of_replicas ]; do
 		sleep 1
 	done
-	# this will help to obtain the hostname of 
+	# this will help to obtain the hostname of
 	# the replicas from the curl
-        unset http_proxy
-        for j in `seq 0 $number_of_attemps`; do
-	        for i in `seq 0 $((number_of_replicas-1))`; do
-		        # this will help that bat does not exit incorrectly 
-			# when curl fails in one of the attemps
-			set +e
-		        REPLICAS[$i]="$(curl $url 2> /dev/null)"
-	                set -e
+	unset http_proxy
```

Comment:
Hi @GabyCT - I know it's awkward, but please can you create a separate commit for all these whitespace changes. It's very hard to see on the github web interface whether you've made any changes to the content of these lines rather than just the formatting.